lib/node/nodes/data-observatory-multiple-measures.js

```diff
@@ -49,17 +49,12 @@ var DataObservatoryMultipleMeasures = Node.create(TYPE, PARAMS, { cache: true, l
             throw new Error('The number of numerators=' + node.numerators.length +
                 ' does not match the number of numerator_timespans=' + node.numerator_timespans.length);
         }
-    }
+    },
 });
 
 module.exports = DataObservatoryMultipleMeasures;
 
-DataObservatoryMultipleMeasures.prototype.sql = function() {
-    var finalColumns = ['_source.*']
-        .concat(this.column_names.map(function(columnName) {
-            return '_data.' + columnName;
-        })).join(', ');
-
+DataObservatoryMultipleMeasures.prototype._composeObsMetaParams = function() {
```

Comment:
I wrote this two month ago and I can't remember why I implemented it in this way. After reviewing a little, it's a really awful implementation, I guess it wasn't one of my best days.... 
 
> Can we express the reduce as a map?

Sure, indeed it's a `map` implemented as `reduce`. :man_facepalming: 

> Are we formatting the JSON within the reduce/map operations? If so, it is better to work with simple objects and format at the end with JSON.stringify, isn't it?

It is and repeating myself: I even don't know what the hell I was thinking while I was doing it.

@ethervoid 🙏  Would you mind fixing that terrible portion of code. I'd be glad to help if needed.